src/date-picker/calendar/header/button/index.tsx

```diff
@@ -9,15 +9,14 @@ import styles from '../../../styles.css.js';
 interface HeaderButtonProps {
   ariaLabel: string;
   isPrevious: boolean;
-  focusable: boolean;
   onChangeMonth: (isPrevious: boolean) => void;
 }
 
-const HeaderButton = ({ ariaLabel, isPrevious, onChangeMonth, focusable }: HeaderButtonProps) => {
+const HeaderButton = ({ ariaLabel, isPrevious, onChangeMonth }: HeaderButtonProps) => {
   const iconName = isPrevious ? 'angle-left' : 'angle-right';
   const additionalAttributes: React.HTMLAttributes<HTMLButtonElement> = {
     className: isPrevious ? styles['calendar-prev-month-btn'] : styles['calendar-next-month-btn'],
-    tabIndex: focusable ? 0 : -1,
+    tabIndex: 0,
```

Comment:
Minor: this line can be removed - the default value of `tabindex` is 0 when not specified.